podman/domain/manager.py

```diff
@@ -2,15 +2,16 @@
 
 from abc import ABC, abstractmethod
 from collections import abc
-from typing import Any, List, Mapping, Optional, TypeVar, Union
+from typing import Any, Optional, TypeVar, Union
+from collections.abc import Mapping
 
 from podman.api.client import APIClient
 
 # Methods use this Type when a subclass of PodmanResource is expected.
 PodmanResourceType: TypeVar = TypeVar("PodmanResourceType", bound="PodmanResource")
 
 
-class PodmanResource(ABC):
+class PodmanResource:
```

Comment:
Why remove ABC base clase?

Reply:
I think I might have been too strict with this one, but my reasoning was the following. The class has no @abstract method so it's not technically an interface. Now that I think about it, we should keep it abstract to prevent the instantiation of the class. I will bring back the ABC inheritance and silence the error on that line